test/runtests.jl

```diff
@@ -1,4 +1,38 @@
-using Test, Example
+using Test
+using DataFrames
+using SQLite
+using MySQL
+using SQLite
+using LibPQ
+using DBConnector
+
+@testset "_dbconnect function for SQLite" begin
+
+    conn= DBConnector._dbconnect(SQLite.DB, "../test/data/sqlite.db")
+    @test @isdefined conn
+    output = DBInterface.execute(conn, "SELECT age FROM PERSON WHERE name = 'John Doe';") |> DataFrame
+    out = output[1,1]
+    expected_output = 30;
+    @test out == expected_output
+
+end
+
+@testset "_dbconnect function for LibPQ" begin
+
+    #conn= DBConnector._dbconnect(LibPQ.Connection,  ENV["POSTGRES_HOST"],ENV["POSTGRES_USER"], ENV["POSTGRES_PASSWORD"], "omop")
+    conn = DBConnector._dbconnect(LibPQ.Connection,  "199.180.155.65", "postgres", "postgres3", "omop")
+    @test @isdefined conn
```

Comment:
Hey Fareeda, could you change these strings to the appropriate GitHub Environment variables?